src/index.js

```diff
@@ -74,7 +74,7 @@ const router = new Router()
                 events_name   : '__disk__',
               }),
               sanitize: input =>
-                input.replaceAll(/[!"#$%&'()*+,.:;<=>?@[\\\]^{|}~]/g, '\\$&'),
+                input.replaceAll(/[!"#$%&'()*+,.:;-<=>?@[\\\]^{|}~]/g, '\\$&'),
```

Comment:
This doesn't work, you missed an escape
```suggestion
                input.replaceAll(/[!"#$%&'()*+,.:;\-<=>?@[\\\]^{|}~]/g, '\\$&'),
```